### PR TITLE
Doc/pox explanations

### PR DESCRIPTION
**Pull request by netrome**

### CODE REVIEW

File: guide.md [hunk 1]
```diff
@@ -71,22 +71,74 @@ Do not confuse burnchain modes **neon** and **helium** with run loops **Neon** a
 ## What is testnet?
 
 ## What is sortition?
+Sortition is the process of selecting a principal to commit a block to the Stacks
```
Comment (igorsyl) [inline guide.md]: I would qualify "principal" with "miner" to highlight that a miner is committing a block.
i.e.: "selecting a miner principal to commit..."
Also, I would draw the distinction between committing a block (which is open to any miner) and selecting a winning block (process which uses sortition)

Comment (igorsyl) [inline guide.md]: Could you add a permalink to the code where sortition is defined?

File: guide.md [hunk 2]
```diff
@@ -71,22 +71,74 @@ Do not confuse burnchain modes **neon** and **helium** with run loops **Neon** a
...
+  tokens the principal has burned.
+- The output of the VRF cannot be predicted before the burn transaction has been
+  included in a block.
+
+***Proof of transfer*** extends *Proof of burn* by instead of burning tokens to candidate,
+miners transfer tokens to addresses of STX holders who participate in stacking. The
+remainder of the sortition process remains the same as in PoB.
+
+## What is a tenure?
+A tenure is the period during which an elected leader propagates transaction data.
```
Comment (igorsyl) [inline guide.md:99]: Could you add a permalink to the code where tenure is defined?

File: guide.md (hunk 1 above)
Comment (netrome) [reply guide.md]: Good point, I will update.

Comment (netrome) [reply guide.md]: Sure, I'll link the section in SIP-001

File: guide.md (hunk 2 above)
Comment (netrome) [reply guide.md:99]: I did not find a definition of tenure in SIP-001, while the term is frequently used throughout the SIP. But I can link the implementation in `testnet/stacks-node/src/tenure.rs`, so I'll do that.

File: guide.md (hunk 1 above)
Comment (netrome) [reply guide.md]: Done!

File: guide.md (hunk 2 above)
Comment (netrome) [reply guide.md:99]: Done.

File: guide.md (hunk 1 above)
Comment (netrome) [reply guide.md]: Done.

File: guide.md (hunk 2 above)
Comment (igorsyl) [reply guide.md:99]: A permalink that is bound to a specific commit may be more reliable than referencing the dynamic master branch where code location can change.

File: guide.md [hunk 3]
```diff
@@ -71,22 +71,83 @@ Do not confuse burnchain modes **neon** and **helium** with run loops **Neon** a
...
+The tenure is terminated when a new burn chain block arrives.
+
+For more details, see the tenure implementation in the stacks blockchain
+[here](https://github.com/stacks-network/stacks-blockchain/blob/master/testnet/stacks-node/src/tenure.rs#L26).
 
 ## What are epochs?
+Epochs in the Stacks blockchain correspond to the leader and leader candidate state
```
Comment (igorsyl) [inline guide.md]: Can we add a link to epoch enum definition in the code?

Comment (igorsyl) [reply guide.md]: Also, I'm confused about this definition for epoch. My understanding is that an epoch is a period in the blockchain's history with defined consensus rules. New epochs are created to accommodate changes in these rules.

Comment (netrome) [reply guide.md]: I will link the definition. Yeah I get the confusion, this description is how epochs are defined in SIP-001 (https://github.com/stacksgov/sips/blob/main/sips/sip-001/sip-001-burn-election.md#definitions). However, consensus-breaking changes (Epoch 2.1, Epoch 3) also reuse the term. I'll update the text to disambiguate the two variants.

File: guide.md (hunk 2 above)
Comment (netrome) [reply guide.md:99]: Good point, I'll update.

Comment (netrome) [reply guide.md:99]: Done.

**Pull request merged**